.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
This settings make VSCode IDE Integration a little more convenient, but serve no other purpose